gammapy/data/observations.py

```diff
@@ -0,0 +1,525 @@
+
+        The dead-time fraction is used in the live-time computation,
+        which in turn is used in the exposure and flux computation.
+    tstart : `~astropy.units.Quantity`
+        Observation start time
+    tstop : `~astropy.units.Quantity`
+        Observation stop time
+    target_radec : `~astropy.coordinates.SkyCoord`
+        Target RA / DEC sky coordinates
+    observatory_earth_location : `~astropy.coordinates.EarthLocation`
+        Observatory location
+    meta : `~collections.OrderedDict`
+        Dictionary to store metadata
+
+    """
+    def __init__(self, obs_id=None, gti=None, events=None, aeff=None, edisp=None, psf=None, bkg=None,
+                 pointing_radec=None, pointing_altaz=None, pointing_zen=None, observation_time_duration=None,
+                 observation_live_time_duration=None, observation_dead_time_fraction=None, tstart=None, tstop=None,
+                 target_radec=None, observatory_earth_location=None, meta=None):
+        self.obs_id = obs_id
+        self.gti = gti
+        self.events = events
+        self.aeff = aeff
+        self.edisp = edisp
+        self.psf = psf
+        self.bkg = bkg
+        self.pointing_radec = pointing_radec
+        self.pointing_altaz = pointing_altaz
+        self.pointing_zen = pointing_zen
+        self.observation_time_duration = observation_time_duration
+        self.observation_live_time_duration = observation_live_time_duration
+        self.observation_dead_time_fraction = observation_dead_time_fraction
+        self.tstart = tstart
+        self.tstop = tstop
+        self.target_radec = target_radec
+        self.observatory_earth_location = observatory_earth_location
+        if not meta:
+            meta = OrderedDict()
+        self.meta = meta
+
+    def __str__(self):
+        """Generate summary info string."""
+        ss = 'Info for OBS_ID = {}\n'.format(self.obs_id)
+        ss += '- Number of events: {}\n'.format(len(self.events.table) if self.events else 'None')
+        ss += '- Number of good time intervals: {}\n'.format(len(self.gti.table) if self.gti else 'None')
+        ss += '- Type of eff. area: {}\n'.format(type(self.aeff))
+        ss += '- Type of energy disp.: {}\n'.format(type(self.edisp))
+        ss += '- Type of PSF: {}\n'.format(type(self.psf))
+        ss += '- Start time: {:.2f}\n'.format(self.tstart.mjd if self.tstart else 'None')
+        ss += '- Pointing pos: RA {:.2f} / Dec {:.2f}\n'.format(
+            self.pointing_radec.ra if self.pointing_radec else 'None',
+            self.pointing_radec.dec if self.pointing_radec else 'None')
+        ss += '- Observation duration: {}\n'.format(self.observation_time_duration)
+        ss += '- Dead-time fraction: {:5.3f} %\n'.format(100 * self.observation_dead_time_fraction)
+
+        return ss
+
+    def peek(self):
+        """Quick-look plots in a few panels."""
+        raise NotImplementedError
+
+    def check_observation(self):
```

Comment:
The `check_observation` method is now copy & pasted and duplicated. Please remove.
Same for `peek`, please remove.

For `__str__`, I see that the two versions on the two obs classes are now different.
It would be better to only keep one, no?
Otherwise things will become hard.

Was there a reason not to introduce a base class that has methods that should be identical to both obs classes, to avoid the copy & paste?